Privacy.md

```diff
@@ -0,0 +1,88 @@
+
+This Service does not use these “cookies” explicitly. However, the app may use third party code and libraries that use “cookies” to collect information and improve their services. You have the option to either accept or refuse these cookies and know when a cookie is being sent to your device. If you choose to refuse our cookies, you may not be able to use some portions of this Service.
+
+## Service Providers
+
+The developer may employ third-party companies and individuals due to the following reasons:
+
+To facilitate our Service;
+
+To provide the Service on our behalf;
+
+To perform Service-related services; or
+
+To assist us in analyzing how our Service is used.
+
+I want to inform users of this Service that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.
+
+## Transfer Of Data
+With these third-party libraries contained in the software, your information, including personal data, may be transferred to computers located outside of your state, province, country or other governmental jurisdiction where the data protection laws may differ than those from your jurisdiction. Your consent to this Privacy Policy followed by your submission of such information represents your agreement to that transfer.
+
+## Security
+
+The developer values your trust in providing us with your Personal Information, thus we are striving to use commercially acceptable means of protecting it. But remember that no method of transmission over the internet, or method of electronic storage is 100% secure and reliable, thus the developer cannot guarantee its absolute security.
+
+## Links to Other Sites
+
+This Service may contain links to other sites. If you click on a third-party link, you will be directed to that site. Note that these external sites are not operated by the developer. Therefore, it is strongly advised that you review the Privacy Policy of these websites. The developer has no control over, and assume no responsibility for the content, privacy policies, or practices of any third-party sites or services.
+
+## Children’s Privacy
+
+This software is intended for general use and has no restrictions on utilization by children except otherwise stated in a law governing that locale.
+
+
+## Changes to This Privacy Policy
+
+The developer reserves the right to make modifications to the policy statement at any time without any notice. This privacy policy may be updated from time to time as the functionalities of the software are further developed and improved and may likely contain disparities that will nullify this current policy. Endeavour to check this document occasionally for any of such changes. 
```

Comment:
There is nothing one has to accept here, and if such a modification was to take place, it would be a GDPR fail right away. This serves no other purpose than to emulate EULAs and scare users away.